### PR TITLE
cli: enhance nav file options

### PR DESCRIPTION
**Pull request by gwbres**
- [x] fix an error when skyplot view is not feasible due to lack of Ground Position definition.
Don't panic and perform further analysis. Notify user they could unlock this visualization
- [x] unlock NAV file visualization with Sv Clock and Clock Drift visualization
- [x] improve doc : exhibit feasible analysis, depending on input context

### CODE REVIEW

File: rinex-cli/README.md [hunk 1]
```diff
@@ -47,14 +47,37 @@ if you know how to operate the preprocessing toolkit
...
-<img align="center" width="450" src="https://github.com/georust/rinex/blob/kepler/doc/plots/skyplot.png">
+- OBS RINEX: all observed physics are depicted.
+See this SSI analysis as an example, which is only one type of physics present
+in OBS RINEX:
+
+<img align="center" width="450" src="https://github.com/georust/rinex/tree/main/doc/plots/esbc00dnk_ssi.png">
+
+- NAV RINEX: satellite vehicle clock data is depicted (bias and drift)
+
+<img align="center" width="450" src="https://github.com/georust/rinex/tree/main/doc/plots/sv_clocks.png">
```
Comment (lnicola) [inline rinex-cli/README.md:66]: Is this link broken?

Comment (gwbres) [reply rinex-cli/README.md:66]: nope, it will unlock when this is merged

Comment (gwbres) [reply rinex-cli/README.md:66]: i'm not sure what's best to manage those cases

Comment (lnicola) [reply rinex-cli/README.md:66]: I don't see `sv_clocks.png` on this branch either, but okay, maybe I'm missing something :-).

**Review by lnicola — APPROVED** [review]

File: rinex-cli/README.md (hunk 1 above)
Comment (gwbres) [reply rinex-cli/README.md:66]: oops correct. *.png are not allowed to be submitted by default, forgot to bypass and introduce this new one

**Pull request merged**